spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/source/SparkTable.java

```diff
@@ -301,7 +393,7 @@ private static CaseInsensitiveStringMap addSnapshotId(CaseInsensitiveStringMap o
       String snapshotIdFromOptions = options.get(SparkReadOptions.SNAPSHOT_ID);
       String value = snapshotId.toString();
       Preconditions.checkArgument(snapshotIdFromOptions == null || snapshotIdFromOptions.equals(value),
-          "Cannot override snapshot ID more than once: %s", snapshotIdFromOptions);
+              "Cannot override snapshot ID more than once: %s", snapshotIdFromOptions);
```

Comment:
stray whitespace change

Reply:
Error operation, Unwanted

Reply:
Looks like the formatting here is still off?

Reply:
fixed